spacy/language.py

```diff
@@ -232,32 +232,6 @@ def meta(self, value):
     def config(self):
         return self._config
 
-    # Conveniences to access pipeline components
-    # Shouldn't be used anymore!
-    @property
-    def tagger(self):
```

Comment:
Are we sure we want to remove these? I don't think they're very harmful, and it keeps old example code working.

Reply:
I think they're kinda harmful, because they produce confusing code that's hard to follow and very intransparent. It also perpetuates this idea of "_the_ tagger" (just like _the_ English model) that we want to move away from. `nlp.get_pipe("tagger")` isn't much more verbose, but much more explicit.

Reply:
Okay fair enough.